tensorflow/core/kernels/tensor_forest/resource_ops.cc

```diff
@@ -114,22 +115,115 @@ class TensorForestTreeSizeOp : public OpKernel {
   }
 };
 
+class TensorForestCreateFertileStatsVariableOp : public OpKernel {
```

Comment:
please add a comment to as what this represents. Eg a resource around statistics of the leaves for all the leaves in a tree that can still be splitted